src/mapnik_vector_tile_render.cpp

```diff
@@ -644,7 +644,7 @@ Napi::Value VectorTile::render(Napi::CallbackInfo const& info)
                     if (lyr.name() == layer_name)
                     {
                         found = true;
-                        layer_idx = idx;
+                        //layer_idx = idx; error: Value stored to 'layer_idx' is never read [clang-analyzer-deadcode.DeadStores,-warnings-as-errors]
```

Comment:
are you sure it is not being read? perhaps clang is wrong and this will create a bug?

Reply:
I agree with clang on this one ^. *Compiler never lies*